REVIEWING.md

```diff
@@ -0,0 +1,108 @@
+
+### Disagreements between reviewers
+
+We do not expect all reviewers to be of the same opinion during code review.
+If you see another reviewer requesting changes to a PR you do not agree with, it is perfectly acceptable to comment to that effect.
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+For all branches except the main branch, this should always remain static.
+PRs into a non-main branch which change the supported Go version should be modified to not require such a change, or rejected if that is not possible.
+Changing supported Go version in the main branch is allowed, but not encouraged.
+Doing so will require significant discussion among maintainers, as it can affect what distributions the main branch can be built.
+
+### Tests
+
+Please Check tests added by the PR.
+If a PR has no new tests, determine if this is actually appropriate - has new functionality been added which should have been tested?
+If the PR does have tests, check to see if they are reasonably comprehensive.
+The Podman Project does not have code coverage standards at present, but we aim to ensure that all new functionality is tested.
+
+### Documentation
+
+All changes to public-facing APIs (e.g. the Podman REST API) and CLI should be appropriately documented.
+API changes require Swagger documentation.
+CLI changes should be documented in the Manpages.
+
+Please validate that PRs making such changes include appropriate documentation.
+Many repositories in the Podman project will enforce this via CI check, but you should still review the contents of the documentation to make sure they are appropriate and complete.
+
+## Things to Avoid
+
+### Bikeshedding and excessively critical reviews
+
+Please avoid bikeshedding during reviews.
+
+Trivial changes that do not affect the ultimate functionality of the PR - for example, small grammar or phrasing changes in documentation, unnecessary renaming of variables - should not block merge of a PR.
+It is acceptable to make such comments, but they should be marked as nice to have changes.
+
+### Asking for unrelated changes
+
+Please do not request that a contributor make significant changes to code their PR did not touch.
```

Comment:
I understand the intention but I am not sure that is the right outcome, some changes might absolutely require some form of larger rework maybe in places they did not touch.
Otherwise you accept hack after hack and the code ends up in a total mess. So if the code is changing something of significant impact and does not seem to be the best implementation maintenance wise then I think it must be valid to block on this.
The end goal of code review should also be to keep the code base maintainable. I do acknowledge what is "maintainable" is totally up to the individual maintainer/reviewer and everybody will have different opinions.

Reply:
Maybe I should put something in here about "best judgement" - relatively minor refactoring is probably fine, but I don't want anyone requesting wide, sweeping changes that will double the size of a PR and introduce a major burden on the contributor.

Reply:
Added some sentences to that effect

Reply:
Thanks, new text sounds good to me.